config/initializers/spam_words.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Decidim::Cdtb::Spam::UserSpamDetector.configure do |config|
+  config.spam_words = %w[viagra sex game free crypto crack xxx luck girls vip download]
+
+  config.spam_regexp = Regexp.union(config.spam_words)
+end
```

Comment:
This does not allow what I meant in my previous comment. How would you overwrite the spam words from the app? repeating this same initializer will expose the internals of the module.

This should only be a `Decidim::Cdtb.configure` with one single value, the `spam_words`. Then we must manage to generate the regexp depending on the value of the `spam_words`. I realize that we even can skip the initializer in the module and have an internal default value in the configuration.

Reply:
In the application an initializer:

```
Decidim::Cdtb.spam_words= ENV['CDTB_SPAM_WORDS']&.split(",")
```
or

```
Decidim::Cdtb.configure do |config|
  config.spam_words= ENV['CDTB_SPAM_WORDS']&.split(",")
end
```

Reply:
Then in this module's engine:
```
config.after_initialize do
  Decidim::Cdtb.config.spam_regexp = Regexp.union(Decidim::Cdtb.config.spam_words)
end
```